README.md

```diff
@@ -14,11 +14,11 @@ Django supports versions 1.4 to 1.8 on Python 2.7 and 3.4.
 
 To install django-downtime::
 
-	pip install django-downtime
```

Comment:
Can you add the indentions back here and below?  They are used to format the text when the README is rendered.